applications/app_ppm.c

```diff
@@ -333,6 +333,34 @@ static THD_FUNCTION(ppm_thread, arg) {
 			}
 			break;
 
+		case PPM_CTRL_TYPE_PID_POSITION_180: // -180 to 180. center ppm safestart
+		case PPM_CTRL_TYPE_PID_POSITION_360: // 0 to +360. minimum ppm safestart
+			if (fabsf(servo_val) < 0.02) {
```

Comment:
Could you drop some comments about what is intended here? Why are you incrementing `pulses_without_power` and why is `0.02` the right value, as opposed to `0.15` or `0.25`?

Reply:
The incrementing pules without power is on every mode. it's part of the safe start feature.
The 0.02 would correspond to 2% of full scale movement. I was having trouble with the default 0.1% range (probably because i'm using a servo tester, not a remote) and I have a "second layer" of safe start, so I increased it to 2%.

Reply:
@kubark42 does this answer your question? This section in the other modes is equally as commented.

Reply:
It does, although I had meant more for the comments to be in the code.